services/api-server/tests/unit/_with_db/test_product.py

```diff
@@ -14,8 +14,11 @@
 from simcore_service_api_server._meta import API_VTAG
 from simcore_service_api_server.models.domain.api_keys import ApiKeyInDB
 
+# pylint: disable=unused-argument
```

Comment:
MINOR: keep all these on the top sorted alphabetically. This way the auto-formatter does not mess them up
I always add these:
```python
# pylint: disable=protected-access
# pylint: disable=redefined-outer-name
# pylint: disable=too-many-arguments
# pylint: disable=unused-argument
# pylint: disable=unused-variable
```

Reply:
Fixed